pkg/extensions/lint/lint.go

```diff
@@ -61,6 +71,13 @@ func (linter *Linter) CheckMandatoryAnnotations(repo string, manifestDigest godi
 	manifestAnnotations := manifest.Annotations
 	for annotation := range manifestAnnotations {
```

Comment:
Why not pick the annotation value from here and pass it to linter.checkIfLogoAnnotationValid() as the parameter?

Also can we add the same logic for lines 109-113?

Reply:
Did the changes. Since the logo annotation is not present in config annotations, I don't see why changes at 109-113 are needed.

Reply:
Because images built using Docker can't set annotations, only labels. So if we want to support images built using Docker we should add support for labels.